tests/playbooks/tests_switch_provider.yml

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# This file was generated by ensure_provider_tests.py
+---
+# set network provider and gather facts
+- hosts: all
+  name: Switch initscripts provider to nm
+  vars:
+    interface: LSR-TST-br34
+  tasks:
+    - name: set fact to use initscripts network_provider
+      set_fact:
+        network_provider: initscripts
+      tags:
+        - always
+    - name: "Create test bridge {{ interface }} via initscripts provider"
+      include_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: "{{ interface }}"
+            state: up
+            type: bridge
+            ip:
+              dhcp4: false
+              auto6: false
+    - include_tasks: tasks/assert_device_present.yml
+    - name: "Remove bridge {{ interface }} via initscripts provider"
```

Comment:
The name says "remove" and the later comment mentions initscripts does not remove the interface. So this should be something like "Take the profile {{ interface }} down via initscripts provider".

Reply:
Fixed. Thanks.